### PR TITLE
PURHASE-198 new registration flow

### PR DESCRIPTION
**Pull request by erikdstock**
#trivial
✅ Needs https://github.com/artsy/eigen/pull/2645 to be merged first. This updates Eigen to use the new registration flow from emission if selected in the debug menu.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -5228,16 +5228,12 @@
...
-				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Italic.otf",
-				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Medium.otf",
-				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-MediumItalic.otf",
-				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Regular.otf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/EBGaramond08-Italic.ttf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/EBGaramond08-Regular.ttf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/EBGaramond12-Italic.ttf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/EBGaramond12-Regular.ttf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/texgyreadventor-bold.ttf",
+				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/texgyreadventor-regular.ttf",
```
Comment (ashfurrow) [inline Artsy.xcodeproj/project.pbxproj]: This is odd... these shouldn't have changed. We can try resetting this file back to its state on master and then re-running `pod install`.

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -5266,20 +5262,17 @@
 				"${PODS_ROOT}/NAMapKit/NAMapKit/pin_red_floating.png",
 				"${PODS_ROOT}/NAMapKit/NAMapKit/pin_red_floating@2x.png",
 				"${PODS_CONFIGURATION_BUILD_DIR}/NPKeyboardLayoutGuide/NPKeyboardLayoutGuide.bundle",
+				"${PODS_CONFIGURATION_BUILD_DIR}/Stripe/Stripe.bundle",
```
Comment (ashfurrow) [inline Artsy.xcodeproj/project.pbxproj]: This change, and the corresponding one below, is okay. It looks like the open source pods got installed?

**Comment by orta** [comment]
Yep cool, 👍

**Pull request merged**